src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml

```diff
@@ -9,9 +9,10 @@
         DiffLineKind.Added => "code-added",
         _ => ""
     };
+
 }
 
-<tr class="code-line" data-line-id="@(isRemoved ? string.Empty : Model.CodeLine.ElementId)">
+<tr class="@(!Model.CodeLine.isDocumentation? "code-line" :"documentation")" data-line-id="@(isRemoved ? string.Empty : Model.CodeLine.ElementId)">
```

Comment:
I don't think we should mark the entire line as doc. We should generate ranges as text `<span>`s and toggle those. And then have a pass that would hide lines that became empty.

This would allow us to hide comments that are only part of the line.